Code/RDGeneral/RDValue-doublemagic.h

```diff
@@ -411,6 +410,22 @@ inline bool rdvalue_cast<bool>(RDValue v) {
   throw boost::bad_any_cast();
 }
 
+// The RDValue has no getKey implementation
+//  so implement our own
+struct KeyIntPair {
+  int key;
```

Comment:
Feels wasteful since the compiler is almost always going to pad this struct out to 16-bytes. Any space left in RDValue for tags large enough to cover common_properties::propnames? That is an optimization that could be done later when a profiler indicates as well.